tests/Feature/Navigation/NavigationTest.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+use Tests\TestCase;
+
+test('pages link appears in navigation', function () {
```

Comment:
Instead of having a NavigationTest file, let's add this test to the WelcomeTest file - which is about testing the home page itself